### ISSUE TITLE
try more advanced architecures

### ISSUE THREAD

**Issue by c-schicho** [c-schicho/ZeroInitialization#3]
us the insights of the initial experiment and try to adopt it to more advanced models and higher dimensional data.

**Comment by c-schicho** [comment]
- [ ] train models on MNIST
- [ ] train a model on CIFAR 10
- [ ] try different learning rates
- [ ] try different optimizer
- [ ] try scaling the random bias
- [ ] try other distributions
- [ ] try with data augmentation
- [ ] try with / without batchnorm
- [ ] try to explain the behavior of the cnn
- [ ] compare test loss / accuracy
- [ ] try different batch sizes (smaller tend to allow for a higher range of good learning rates)
